app/assets/stylesheets/quiz.scss

```diff
@@ -0,0 +1,100 @@
+/* Hide the Ordinary Checkbox */
+input[type="checkbox"] {
+    display: none;
+}
+
+input[type='checkbox'] + label{
+  position: relative;
+  font-family: "Varela Round", -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji";
```

Comment:
When everything is set and the application works the way we want, we should go back and check to make sure that everything follows a general font formation.